libs/base/ui/overlays/modal/src/modal.styles.ts

```diff
@@ -60,4 +60,9 @@ export const styles = css`
   oryx-icon-button:first-child button {
     padding: 0;
   }
+
+  ::slotted(oryx-button[full-width]),
+  oryx-button[full-width] {
+    width: 100%;
+  }
```

Comment:
I guess we can use just CSS to make the button full width without using extra full-width atrrbute
```
:host([footerButtonFullWidth]) footer button,
:host([footerButtonFullWidth])::slotted(button) {
  width: 100%;
}

Reply:
Also, we can set the gap between buttons if there will be more than one button

Reply:
I like this idea too, seems like an easier way to achieve the same visuals.

Reply:
Seems the footer already supplies a gap.